### PR TITLE
web workers, file upload

### PR DESCRIPTION
**Pull request by 3xtr4t3rr3str14l**
-web worker for loading ssz types
-web worker for serialization and hashTreeRooting
-ability to upload input type values via a file

### CODE REVIEW

**Comment by wemeetagain** [comment]
Can we also add an upload option on the serialize side? Eg: I'd like to upload a .json or .yaml file to be serialized

File: package.json [hunk 1]
```diff
@@ -10,15 +10,16 @@
     "dev": "cross-env NODE_ENV=development webpack-dev-server --color --progress",
     "build": "cross-env NODE_ENV=production webpack --color --progress",
     "start": "serve dist",
-    "lint": "eslint --ext .tsx src/"
+    "lint": "eslint --ext .tsx src/ --fix"
```
Comment (wemeetagain) [inline package.json:13]: can you add an additional script for lint fix instead of editing the `lint` script?

File: .eslintrc.js [hunk 2]
```diff
@@ -50,17 +48,13 @@ module.exports = {
     }],
     "camelcase": "error",
     "func-call-spacing": "off",
-    "max-len": ["error", {
```
Comment (wemeetagain) [inline .eslintrc.js:53]: just set to a warning

File: src/components/worker.js [hunk 3]
```diff
@@ -1,32 +1,33 @@
+import {presets} from "../util/types";
```
Comment (wemeetagain) [inline src/components/worker.js:1]: why is this file a .js file now?

File: package.json (hunk 1 above)
Comment (3xtr4t3rr3str14l) [reply package.json:13]: yes, will put in next commit

File: .eslintrc.js (hunk 2 above)
Comment (3xtr4t3rr3str14l) [reply .eslintrc.js:53]: yes, will put in next commit

File: src/components/worker.js (hunk 3 above)
Comment (3xtr4t3rr3str14l) [reply src/components/worker.js:1]: it's always been a .js file, but good catch.  i'll change it to .tsx

File: src/components/worker.tsx [hunk 4]
```diff
@@ -1,32 +1,33 @@
+import {presets} from "../util/types";
 import {BasicType, CompositeType, isBooleanType, isListType, isVectorType, isBitListType, isBitVectorType, isContainerType, isByteVectorType, isNumberUintType, isBigIntUintType} from '@chainsafe/ssz';
 
-function randomNumber(length: number): number {
```
Comment (wemeetagain) [inline src/components/worker.tsx:3]: Looks like some of the function input/output types were lost in this file from ts -> js -> ts

Comment (3xtr4t3rr3str14l) [reply src/components/worker.tsx:3]: yeah, forgot to add back the input types.  should all be there now.

**Pull request merged**